### PR TITLE
Scaffold single exhibition

### PR DESCRIPTION
**Pull request by mikeyrayvon**
`yarn install`?

adds masonry, imagesloaded, and slick-carousel

images for testing are in `assets/dummy/ramiro`

### CODE REVIEW

File: single-exhibition.php [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+          if (!empty($images)) {
+        ?>
+        <div class="masonry-holder">
+          <div class="masonry-gutter"></div>
+          <?php
+            foreach($images as $image) {
+          ?>
+          <div class="masonry-item carousel-trigger">
+            <?php
+              echo wp_get_attachment_image($image['image_id'], 'full', false, array('class'=>'carousel-trigger','data-no-lazysizes'=>'true'));
```
[the remark this answers is not in the document]
Comment (mikeyrayvon) [reply single-exhibition.php]: lazy load prevents masonry from laying out correctly

File: src/js/main.js [hunk 2]
```diff
@@ -97,9 +110,81 @@ class Site {
...
   }
+
+  bindCarouselToggles() {
+    var _this = this;
+
+    if ($('.carousel-trigger').length) {
+      $('.carousel-trigger').on('click', function() {
+        _this.scrollOffset = _this.$window.scrollTop();
+
+        _this.$mainContainer.css('top', '-' + _this.scrollOffset + 'px');
```
[the remark this answers is not in the document]
Comment (mikeyrayvon) [reply src/js/main.js]: animate what? this is to make the content not scroll while the overlay is open

File: src/styl/responsive/mobile.styl [hunk 3]
```diff
@@ -1,4 +1,12 @@
 $border-style = 1px solid $color-black
+$carousel-arrow-diameter = 50px
+
+html, body
+  height:100%
```
[the remark this answers is not in the document]
Comment (mikeyrayvon) [reply src/styl/responsive/mobile.styl]: for this overflow thing to work

**Comment by mikeyrayvon** [comment]
sorry for being lazy about commenting :/

File: src/js/main.js [hunk 4]
```diff
@@ -137,32 +141,54 @@ class Site {
...
+          _this.$body.addClass('carousel-active');
+        }
       });
 
-      $('.carousel-overlay').on('click', function(e) {
+      $('#carousel-overlay').on('click', function(e) {
         if (!$(e.target).hasClass('slick-arrow')) {
           _this.$body.removeClass('carousel-active');
 
+          // reset scroll position
```
Comment (varffs) [inline src/js/main.js]: works now :)

**Pull request merged**